packages/grid/src/styles/column.module.css

```diff
@@ -0,0 +1,688 @@
+:root {
+  --column-gap: var(--column-gap);
+  --column-width: calc((100% - (11 * var(--column-gap))) / 12);
+}
+
+@media (min-width: 768px) {
+  :root {
+    --column-gap: var(--gap-s);
+  }
+}
+
+@media (min-width: 1024px) {
+  :root {
+    --column-gap: var(--gap-m);
+  }
+}
+
+@media (min-width: 1440px) {
+  :root {
+    --column-gap: var(--gap-xl);
+  }
+}
+
+/* Small Width Classes */
+.sm-1 {
+  width: var(--column-width);
+}
+
+.sm-2 {
+  width: calc((var(--column-width) * 2) + var(--column-gap));
+}
+
+.sm-3 {
+  width: calc((var(--column-width) * 3) + (2 * var(--column-gap)));
+}
+
+.sm-4 {
+  width: calc((var(--column-width) * 4) + (3 * var(--column-gap)));
+}
+
+.sm-5 {
+  width: calc((var(--column-width) * 5) + (4 * var(--column-gap)));
+}
+
+.sm-6 {
+  width: calc((var(--column-width) * 6) + (5 * var(--column-gap)));
+}
+
+.sm-7 {
+  width: calc((var(--column-width) * 7) + (6 * var(--column-gap)));
+}
+
+.sm-8 {
+  width: calc((var(--column-width) * 8) + (7 * var(--column-gap)));
+}
+
+.sm-9 {
+  width: calc((var(--column-width) * 9) + (8 * var(--column-gap)));
+}
+
+.sm-10 {
+  width: calc((var(--column-width) * 10) + (9 * var(--column-gap)));
+}
+
+.sm-11 {
+  width: calc((var(--column-width) * 11) + (10 * var(--column-gap)));
+}
```

Comment:
Please use for recurring rules  : https://www.npmjs.com/package/postcss-each or https://www.npmjs.com/package/postcss-for and https://github.com/postcss/postcss-mixins as well